src/OnboardingSPA/components/ErrorState/Step/SiteGen/Migration/index.js

```diff
@@ -0,0 +1,85 @@
+import { useEffect } from '@wordpress/element';
```

Comment:
Shall we reuse the existing error state? This one seems to be almost similar to the other one, and it does not handle mobile view, if I am not mistaken.

Reply:
It handles mobile view, I will check how can re-use the existing error state for migration as well.